src.test.resources/testng.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove the commented code, again its a general note, applied everywhere on code. We always send PR after removing commented code